samples/snippets/detect_intent_texts_with_location.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DialogFlow API Detect Intent Python sample to use regional endpoint.
+
+Examples:
+  python detect_intent_texts_with_location.py -h
+  python detect_intent_texts_with_location.py --project-id PROJECT_ID \
+  --location-id LOCATION_ID --session-id SESSION_ID \
+  "hello" "book a meeting room" "Mountain View"
+"""
+
+import argparse
+import uuid
+
+
+# [START dialogflow_detect_intent_text_with_location]
```

Comment:
Putting the region tag outside of the method will make the sample render with the method name/def. Is that intended? 

For instance https://github.com/googleapis/python-bigquery/blob/master/samples/snippets/authenticate_service_account.py shows how you might just want the content of the method, not the method itself.

Reply:
I think that's intended, I followed the current style of other samples.